packages/core-utils/src/time.js

```diff
@@ -63,8 +62,9 @@ export function formatDurationWithSeconds(seconds) {
  * @returns {string} formatted text representation
  */
 export function formatTime(ms, options) {
-  return moment(ms + (options && options.offset ? options.offset : 0)).format(
-    options && options.format ? options.format : OTP_API_TIME_FORMAT
+  return format(
+    ms + (options && options.offset ? options.offset : 0),
+    options && options.format ? options.format : OTP_API_DATE_FORMAT_DATE_FNS
```

Comment:
This is the `formatTime` method, so this should still use `OTP_API_TIME_FORMAT`:

```suggestion
    options && options.format ? options.format : OTP_API_TIME_FORMAT
```

Reply:
Very good catch thank you! Resolved in 13999c397c4642208ed8f91450f11f5c740d7846